Project-II-Calculator/calculator/src/components/DisplayComponents/Display.css

```diff
@@ -0,0 +1,72 @@
+.calculatorDisplay {
+  width: 360px;
+  height: 420px;
+  display: flex;
+  flex-direction: column;
```

Comment:
```suggestion

```
Just a reminder that elements will align in a column by default, you only need to use this if you also need other flex properties